gufe/mapping/__init__.py

```diff
@@ -1,7 +0,0 @@
-# This code is part of gufe and is licensed under the MIT license.
-# For details, see https://github.com/OpenFreeEnergy/gufe
-"""Defining the relationship between different components"""
-from .componentmapping import ComponentMapping
```

Comment:
Similarly, this can't go away until 2.0

Reply:
hm would it make sense to split this out into a new PR, or drop it completly? I m not sure of your intend here.

Reply:
I think now it should be not api breaking.